pkg/processor/trigger/trigger.go

```diff
@@ -347,22 +347,22 @@ func (at *AbstractTrigger) SubscribeToControlMessageKind(kind controlcommunicati
 
 // UnsubscribeFromControlMessageKind unsubscribes all workers from control message kind
 func (at *AbstractTrigger) UnsubscribeFromControlMessageKind(kind controlcommunication.ControlMessageKind,
-	controlMessageChan chan *controlcommunication.ControlMessage) error {
-
+	controlMessageChan chan *controlcommunication.ControlMessage) {
 	at.Logger.DebugWith("Unsubscribing channel from control message kind",
 		"kind", kind,
 		"numWorkers", len(at.WorkerAllocator.GetWorkers()))
 
 	for _, workerInstance := range at.WorkerAllocator.GetWorkers() {
 		if err := workerInstance.Unsubscribe(kind, controlMessageChan); err != nil {
-			return errors.Wrapf(err,
-				"Failed to unsubscribe channel from control message kind %s in worker %d",
-				kind,
-				workerInstance.GetIndex())
+			at.Logger.WarnWith("Failed to unsubscribe channel from control message",
+				"kind", kind,
+				"worker_id", workerInstance.GetIndex(),
+				"error", err)
 		}
 	}
-
-	return nil
+	at.Logger.DebugWith("Successfully unsubscribed channel from control message kind",
```

Comment:
```suggestion
	at.Logger.DebugWith("Successfully unsubscribed workers from control message kind",
```

Reply:
@TomerShor why? We unsubscribe specific channel from all workers control messages(not a worker), we can change it to `"Successfully unsubscribed from control message kind"`

Reply:
Yeah let's go with "Successfully unsubscribed from control message kind"